tests/test_util/test_table.py

```diff
@@ -1275,17 +1276,18 @@ def test_head(self):
         display = table.display
         head = TrapOutput()
```

Comment:
this is awesome, I think it means you can delete the entire TrapOutput class. Can you delete it and all uses and replace with this contextlib approach.

Reply:
Not a problem at all! thank you Gavin!